app.rb

```diff
@@ -1,8 +1,38 @@
-require 'sinatra/base'
+require "sinatra/base"
+require "sinatra/reloader"
+require "./lib/peep"
 
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  get "/test" do
```

Comment:
You could remove your test page when you're finished to clean up this file if it's not going to be used again